edx/frontend-app-gradebook
# Gradebook Test Plan

**TL;DR -** Add a first-pass at a test plan.

JIRA: [EDUCATOR-5712](https://openedx.atlassian.net/browse/EDUCATOR-5712)

**What changed?**

- Adds `test-setup.md` for basic course/feature/track setup needed for testing Gradebook.
- Enumerates the basic workflows and provides steps to test/verify.
- Adds reminder to the _Developer Checklist_ to update documentation.

**Developer Checklist**

**Testing Instructions**

- Verify workflows and setup instructions. I suggest viewing the "Rich Diffs" for files so you can see what they will look like in rendered Markdown.
- Critique any parts that are unclear.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:



FYI: @edx/masters-devs-gta
